onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -195,13 +195,11 @@ class PlannerImpl {
   // Find if there exists some input tensor that we can use in-place for output_arg
   bool FindReusableInput(const onnxruntime::Node& node, int output_arg_num, MLValueIndex* reusable_input) {
     auto p_output_arg = node.OutputDefs()[output_arg_num];
-    auto p_opkernel_def = utils::GetKernelDef(kernel_registry_, node);
+    const KernelCreateInfo* ci;
+    Status st = kernel_registry_.SearchKernelRegistry(node, &ci);
+    if (!st.IsOK() || ci == nullptr || ci->kernel_def == nullptr) return false;
 
```

Comment:
nit: I know the google c++ style allows this, but putting the return statement on the same line as the 'if' means you have to reformat the code to put a breakpoint on the 'return' statement if you wanted to debug when the return was used.